### PR TITLE
Fix false positive on the CI and missed bugs

### PR DESCRIPTION
**Pull request by phoracek**
**What this PR does / why we need it**:

Fix operator-sdk test --namespace flag. It has been changed to --operator-namespace. Using --namespace cause operator-sdk to return immediately with false positive.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
After some `git bisect`ing - it seems that the test failing : `should report non-empty list of deployed container` issue was introduced in [Bump kubernetes-nmstate 0.21.0 (#403)](https://github.com/kubevirt/cluster-network-addons-operator/commit/e7e4a3ca84415a7baf154e2e13bb30492b72bb10)

@qinqon - can you take a look why it fails?

File: test/e2e/workflow/validation_test.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ var _ = Describe("NetworkAddonsConfig", func() {
 				NMState:     &opv1alpha1.NMState{},
 			}
 			CreateConfig(configSpec)
-			CheckConfigCondition(ConditionAvailable, ConditionTrue, time.Minute, CheckDoNotRepeat)
+			CheckConfigCondition(ConditionAvailable, ConditionTrue, 2*time.Minute, CheckDoNotRepeat)
```
Comment (qinqon) [inline test/e2e/workflow/validation_test.go:47]: Is this related ?

**Comment by qinqon** [comment]
Both jobs are not false possitive anymore, let's approve this

/lgtm
/approve
